dev/how-to/deploy/data-migration-with-ansible.md

```diff
@@ -316,7 +316,7 @@ VjX8cEeTX+qcvZ3bPaO4h0C80pe/1aU=
 
 ## Step 8: Edit variables in the `inventory.ini` file
 
-This step shows how to edit the variable of the deployment directory, how to configure the relay log synchronization position and the relay log GTID synchronization mode, and explains the global variables in the `inventory.ini` file.
+This step shows how to edit the variable of the deployment directory, how to configure the relay log replication position and the relay log GTID replication mode, and explains the global variables in the `inventory.ini` file.
```

Comment:
We would normally just say "relay log position". I was looking to provide a suggestion on how to improve it, but it is quite repetitive with the text below. Maybe for this initial summary just say this:
```suggestion
This step shows how to make configuration changes to the `inventory.ini` file.
```